R/shinyui.R

```diff
@@ -38,7 +40,10 @@ renderPage <- function(ui, showcase=0, testMode=FALSE) {
     # Put the body into the default template
     ui <- htmlTemplate(
       system.file("template", "default.html", package = "shiny"),
-      body = ui
+      lang = lang,
+      body = ui,
+      # this template is a complete HTML document
+      document_ = TRUE
```

Comment:
Just checking: the template previously was autodetected so that this behaved like it was TRUE, right?

Reply:
Correct. I'll be removing this line though

Reply:
I'm going to keep this line in.

Prior behavior had `document_ = "auto"` by default.  It would hit these lines of code in `htmltools`: https://github.com/rstudio/htmltools/blob/f589c77bc941428d4bf2cec3ca5f15399ce18f47/R/template.R#L63-L65 where `htmltools` would test for some form of `<html>` to set `document_`.

To make sure the `lang` logic does not corrupt the template `document_` logic, I am explicitly setting `document_ = TRUE`.